### ISSUE TITLE
Warn (or do something better) when the project path is on a symlink.

### ISSUE THREAD

**Issue by technosophos** [Masterminds/glide#363]
There is a long-standing issue where symlinks on the GOPATH can cause the import scanner to get confused. We should at least warn someone when we detect this is the case.

**Comment by rbuckland** [comment]
+1

I would go slightly further - until you are comfortable where the problem actually lies / and has been subsequently resolved. I would go so far as to error right out of the build; except allow ``--ignore-sym-link-detection`` flag.

**Comment by sdboyer** [comment]
you can assign this one to me, if ya want

**Comment by thockin** [comment]
Is this manifesting as a real issue?  We use symlinks in various ways to get compat across older and newer Go versions and so far it seems to work OK.  Please don't break it if you don't have to.

**Comment by sdboyer** [comment]
@thockin yeah, it's brought regularly into the channel, though it hasn't manifested much as an issue in the queue (i think?)

I don't think we could safely add anything more than a warning here, anyway.
